applications/sim_showers_for_trigger_rates.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/python3
```

Comment:
In general, I would advocate for full names of what applications are doing (e.g., this one, I would call simulate_showers_for_trigger_rates), otherwise we might have very soon difficulties in understanding what the executables are doing (very minor comment)

Reply:
In general, that is what we are doing. We are using long and clear names everywhere. In this case, I thought 'sim' was fine because we will have multiple applications of this kind, so this would be a clear pattern.
But I don't mind rename it. I can do it before merging if you want that.